src/components/Maps/Marker.tsx

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable */
```

Comment:
same deal here - this shouldn't be necessary, and if it's unavoidable it should be disabled on a line-by-line basis not for the entire file